app/models/status_edit.rb

```diff
@@ -45,6 +45,10 @@ class PreservedMediaAttachment < ActiveModelSerializers::Model
   delegate :local?, :application, :edited?, :edited_at,
            :discarded?, :visibility, :language, to: :status
 
+  def with_media?
```

Comment:
There's a few helpers like these which are on the `Status` model, but not present on the `StatusEdit` model, the other methods I know of are: `with_poll?` and `with_preview_card?`, these methods missing make it difficult to reuse the status content partial between both statuses and status edits.